ui: fixed statements pagination

vladlos opened the pull request:
hide the pagination when statements show a single page

Resolves: #46264

Release justification: bug fix

Release note (ui): none

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/46312)
<!-- Reviewable:end -->

dhartunian commented:
@elkmaster LGTM, please add a release justification to the commit message and PR message. Thanks!

dhartunian commented:
@elkmaster sorry I should have clarified how the Release justifications work, they are not a re-statement of the commit message. You pick from a few options (they're in the commit template):

```
# Categories for release justification:
#     - non-production code changes
#     - bug fixes and low-risk updates to new functionality
#     - fixes for high-priority or high-severity bugs in existing functionality
#     - low risk, high benefit changes to existing functionality
#
```

In this case, this falls into the second category so you would add something like:

```
Release justification: bug fix
```

The goal of the Release Justification is to capture some reasoning around why something is being merged during the stability period. Let me know if you have any further questions, thanks!

vladlos committed:
ui: fixed statements pagination

hide the pagination when statements show a single page

Resolves: #46264

Release justification: bug fix

Release note (ui): none

dhartunian commented:
bors r+

dhartunian commented:
bors r+
